addok_psql_store/__init__.py

```diff
@@ -43,28 +43,25 @@ def upsert(self, *docs):
         INSERT INTO {PG_TABLE} (key, data) VALUES %s
             ON CONFLICT DO NOTHING
         '''.format(**config)
-        with self.conn.cursor() as curs:
+        with self.pool.getconn() as conn, conn.cursor() as curs:
             execute_values(curs, insert_into_query, docs)
-        self.conn.commit()
 
     def remove(self, *keys):
         delete_from_query = '''
         DELETE FROM {PG_TABLE} WHERE key=%s
         '''.format(**config)
-        with self.conn.cursor() as curs:
+        with self.pool.getconn() as conn, conn.cursor() as curs:
             curs.executemany(delete_from_query, (keys, ))
-        self.conn.commit()
 
     def flushdb(self):
         drop_table_query = '''
         DROP TABLE IF EXISTS {PG_TABLE}
         '''.format(**config)
-        with self.conn.cursor() as curs:
+        with self.pool.getconn() as conn, conn.cursor() as curs:
             curs.execute(drop_table_query)
-        self.conn.commit()
```

Comment:
Beware that removing `commit()` everywhere lead to loss of data when I tried loading a whole county. Might be because of the lack of pool though.